lib/rex/proto/ssh/connection.rb

```diff
@@ -83,15 +83,16 @@ def initialize(io = nil, options = self.default_options, context = {})
     @context = context
     @logger = Logger.new self.class.name
```

Comment:
Unrelated to my PR; this logs to a random file called `Rex::Proto::Ssh::Connection` which outputs info such as:

```
I, [2022-08-04T23:20:52.527254 #37443]  INFO -- Rex::Proto::Ssh::Connection[3d18]: start connection
I, [2022-08-04T23:20:52.532739 #37443]  INFO -- Rex::Proto::Ssh::Connection[3d18]: start connection loop
I, [2022-08-04T23:20:52.532998 #37443]  INFO -- Rex::Proto::Ssh::Connection[3d18]: received SSH_MSG_CHANNEL_OPEN
D, [2022-08-04T23:20:52.533096 #37443] DEBUG -- HrrRbSsh::Message::SSH_MSG_CHANNEL_OPEN[3d2c]: decoded message: {:"message number"=>90, :"channel type"=>"session", :"sender channel"=>0, :"initial window size"=>2097152, :"maximum packet size"=>32768}
D, [2022-08-04T23:20:52.533367 #37443] DEBUG -- HrrRbSsh::Connection::Channel[3d40]: in start: nil
D, [2022-08-04T23:20:52.533659 #37443] DEBUG -- HrrRbSsh::Message::SSH_MSG_CHANNEL_OPEN_CONFIRMATION[3d68]: encoding message: {:"message number"=>91, :"channel type"=>"session", :"recipient channel"=>0, :"sender channel"=>0, :"initial window size"=>100000, :"maximum packet size"=>100000}
I, [2022-08-04T23:20:52.533458 #37443]  INFO -- HrrRbSsh::Connection::Channel[3d40]: start out sender thread
```

This should probably be wired up to use a framework compatible logger instead, so they appear in `~/.msf4/logs/framework.log`